tests/unit/amo/components/TestSearchFormSuggestion.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import Suggestion from 'amo/components/SearchForm/Suggestion';
+import Icon from 'ui/components/Icon';
+import { fakeAddon } from 'tests/unit/amo/helpers';
+
+
+describe(__filename, () => {
+  const shallowComponent = (props) => {
```

Comment:
There is also a `render()` function in Enzyme but it does different things than `shallow()`.